### DISCUSSION TITLE
[Q/A][Java SDK (NacosRegistry)] notifySubscriber方法为什么是调用NacosAggregateListener#saveAndAggregateAllInstances方法获取所有的instances

### DISCUSSION THREAD

**Discussion by lossend** [apache/dubbo#14191] [Question]
### Pre-check

- [ ] I am sure that all the content I provide is in English.

### Apache Dubbo Component

Java SDK (apache/dubbo)

### Details

在notifySubscriber方法为什么是调用NacosAggregateListener#saveAndAggregateAllInstances方法获取所有的instances，拿到的是所有的服务实例，而不是跟这个serviceName相关的instances
![image](https://github.com/apache/dubbo/assets/49791601/7e8ec135-9020-4702-a2af-da168be88e21)

### Code of Conduct

- [ ] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by AlbumenJ** [comment]
Please submit the discussion in English :)
